lib/src/ui/movies_details_page.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import '../widgets/detailCardsWidgets/custom_appbar.dart';
+import '../widgets/detailCardsWidgets/movie_resume.dart';
+import '../model/movie_result.dart';
+import '../widgets/detailCardsWidgets/movie_main_info.dart';
+
+class MovieDetailsPage extends StatelessWidget {
+  const MovieDetailsPage(
+    this.data, {
+    Key? key,
+  }) : super(key: key);
+
+  final MoviesResult data;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: <Widget>[
+          CustomAppBar(
+            posterPath: data.posterPath,
+          ),
+          SliverList(
+            delegate: SliverChildListDelegate(
+              [
+                SizedBox(
```

Comment:
Remember what we talked about const Widgets

Reply:
right , done.